circuit/tx_constraints.go

```diff
@@ -418,8 +430,10 @@ func VerifyTransaction(
 	)
 	newStateRoot := hFunc.Sum()
 	types.IsVariableEqual(api, notEmptyTx, newStateRoot, tx.StateRootAfter)
-	types.IsVariableEqual(api, isEmptyTx, tx.StateRootBefore, tx.StateRootAfter)
-	return isOnChainOp, pubData, nil
+
+	roots[0] = NewAccountRoot
```

Comment:
`NewAccountRoot` is a tmp variable, can we use lower case?

Reply:
So is `NewNftRoot`

Reply:
fixed